BUILDING.md

```diff
@@ -253,6 +253,23 @@ To install this version of Node.js into a system directory:
 $ [sudo] make install
 ```
 
+#### Building a debug build
+
+If you run into a issue where Node.js segfaults and the "segfault-handler" does not provide enough information, you might have to build a debug build of Node.js to get enough information to debug the issue further, this is done by :
+
+```console
+$ ./configure --debug
+$ make -j4
```

Comment:
I might suggest `make -C out BUILDTYPE=Debug -j4` to avoid unnececerily building the `Release` binary. That could allow you to replace the following paragraph with:
```
If the build finished successfully, the debug enabled binary will be at `out/Debug/node`
```